### ISSUE TITLE
Enable Windows

### ISSUE THREAD

**Issue by rboston628** [rboston628/THRAIN#21]
THRAIN was developed on macOS using gcc, and not a lot of thought was put into portability.

When attempting to set up Windows runners in PR #7, it was revealed THRAIN will not compile in Windows, due to reliance on functionality only present in POSIX.

The following are compiler errors present when compiling on Windows:

- [ ] `M_PI` inside `chandra.cpp`, should be the `m_pi` defined inside `constants.h`.
- [ ] `geline` inside `SimpleWD.cpp`, line 269, maybe replace with `std::fgets`?
``` bash
src/STARS/SimpleWD.cpp:269:21: error: 'getline' was not declared in this scope; did you mean 'std::getline'?
  269 |         line_size = getline(&input_buffer, &buffer_size, input_file);
      |                     ^~~~~~~
      |                     std::getline
```

**Comment by rboston628** [comment]
See PR #20

**Comment by rboston628** [comment]
Closed by PR #20
